File: mpp/LabMPPCurse/Models/src/main/java/Models/Personal/Personal.java
package Models.Personal;

import Models.HasId;

public class Personal implements HasId<Integer> {

    private int id;
    private String userName;
    private String pass;
    private String nume;
    private String prenume;

    public Personal(String username, String password) {
        this.userName=username;
        this.pass=password;
    }

    @Override
    public Integer getId() {
        return this.id;
    }

    @Override
    public void setId(Integer integer) {
        this.id=integer;
    }

    public Personal(int id, String userName, String pass, String nume, String prenume) {
        this.id = id;
        this.userName = userName;
        this.pass = pass;
        this.nume = nume;
        this.prenume = prenume;
    }

    @Override
    public String toString() {
        return "Personal{" +
                "id=" + id +
                ", userName='" + userName + '\'' +
                ", pass='" + pass + '\'' +
                ", nume='" + nume + '\'' +
                ", prenume='" + prenume + '\'' +
                '}';
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPass() {
        return pass;
    }

    public void setPass(String pass) {
        this.pass = pass;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }
}

File: mpp/CurseMPP/CurseMPP/Models/Personal/Personal.cs
namespace CurseMPP.Models
{
	public class Personal : HasId<int>
	{

		public Personal(int id, string user, string pass, string numes, string prenumes)
		{
			Id = id;
			userName = user;
			password = pass;
			nume = numes;
			prenume = prenumes;
		}


		public int Id { get; set; }
		public string userName { get; set; }
		public string password { get; set; }
		public string nume { get; set; }
		public string prenume { get; set; }

		public override string ToString()
		{
			return "id: " + Id + " userName: " + userName + " password: " + password + " nume: " + nume + " prenume: " + prenume;
		}
	}
}